src/Database/Connection.php

```diff
@@ -158,10 +159,11 @@ public function driver($driver = null, $config = [])
             return $this->_driver;
         }
         if (is_string($driver)) {
-            if (!class_exists($driver)) {
+            $className = App::className($driver, 'Database/Driver');
+            if (!$className || !class_exists($className)) {
```

Comment:
That would work for sure, given that it's suitable for `App::className()` to check possible external names.